support/origintracking/svdraw2/svdraw2.py

```diff
@@ -0,0 +1,258 @@
+import os, sys
```

Comment:
Not going to try reviewing this, but I trust that you (mostly) know what you are doing :-)

Also why svdraw2 and not svdraw?

Reply:
Oh yeah, this script is a total disgusting hack. I offer no real defense other than that it's very very useful (see the docs PR for examples of the diagrams it makes.)

It'd be nice to have a not-horrible version of a viz tool like this. It could probably be built entirely in silver using reflection, although it might be kind of a PITA to do the graph processing in silver. It might make more sense to just rewrite a not-horrible python version.

Reply:
Hmm, maybe.  Silver does have a native graph library (under `silver:util:raw:graph`) by the way.